Countly.js

```diff
@@ -181,25 +169,10 @@ Countly.demo = function(){
 
 Countly.setLocation = function(countryCode, city, location, ipAddress){
     var args = [];
-    args.push(countryCode || "");
-    args.push(city || "");
-    if(!location){
-        location = "0.0,0.0";
-    }
-    var locationArray = location.split(",")
-    var newStringLocation = "";
-    if(locationArray[0].indexOf(".") == -1){
-        newStringLocation = newStringLocation+""+parseFloat(locationArray[0]).toFixed(2);
-    }else{
-        newStringLocation = newStringLocation+""+locationArray[0]
-    }
-    if(locationArray[1].indexOf(".") == -1){
-        newStringLocation = newStringLocation+","+ parseFloat(locationArray[1]).toFixed(2);
-    }else{
-        newStringLocation = newStringLocation+","+locationArray[1]
-    }
-    args.push(newStringLocation);
-    args.push(ipAddress || "0.0.0.0");
+    args.push(countryCode || "null");
```

Comment:
what exactly is `null` and why a `String` of type is defined?

Reply:
As far as I understand, the "null" thing is the workaround of how to pass a null value over to the native side. Maybe a limitation of the argument array.